mobile/src/lib/crash-reporter.js

```diff
@@ -0,0 +1,15 @@
+/* global __SENTRY_TOKEN__ */
+import Raven from 'raven-js'
+
+export function logException (err, context) {
+  if (!Raven.isSetup()) {
+    Raven.config(`https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`).install()
```

Comment:
This were discussed in #118 . We decide to put the variable part of the sentry url in webpack, and to let the url in the code.